animated-gif/src/main/java/com/facebook/animated/gif/GifFrame.java

```diff
@@ -81,6 +81,10 @@ public boolean hasTransparency() {
     return nativeHasTransparency();
   }
 
+  public int getTransparentPixel() {
```

Comment:
Can you call this `getTransparentPixelColor()` to make it clear that this returns a color?